frame.go

```diff
@@ -205,6 +205,22 @@ func ParseConsistency(s string) Consistency {
 	}
 }
 
+// ParseConsistencyWrapper wraps gocql.ParseConsistency to provide an err
+// return instead of a panic
+func ParseConsistencyWrapper(s string) (consistency Consistency, err error) {
+	defer func() {
```

Comment:
Yea, was thinking the same, but orders are orders ;) Based on the comment, I thought that this is what you guys want.